apps/web/pages/settings/organizations/platform/oauth-clients/components/OAuthClientForm.tsx

```diff
@@ -128,13 +162,24 @@ export const OAuthClientForm: FC = () => {
           />
         </div> */}
         <div className="mt-6">
-          <TextField label="Redirect uri" required={true} {...register("redirectUri")} />
+          <TextField type="url" label="Redirect uri one" required={true} {...register("redirectUriOne")} />
+        </div>
+        <div className="mt-6">
+          <TextField type="url" label="Redirect uri two" {...register("redirectUriTwo")} />
         </div>
         <div className="mt-6">
-          <h1 className="text-base font-semibold underline">Permissions</h1>
+          <TextField type="url" label="Redirect uri three" {...register("redirectUriThree")} />
+        </div>
+        <div className="mt-6">
+          <div className="flex justify-between">
+            <h1 className="text-base font-semibold underline">Permissions</h1>
+            <Button type="button" onClick={selectAllPermissions}>
+              Select all
```

Comment:
When all are selected the "Select all" should change to de-select all instead.

<img width="889" alt="Screenshot 2024-02-27 at 10 21 04" src="https://github.com/calcom/cal.com/assets/42170848/f9df29b9-7a23-4df1-bba9-f6e90c8a7de5">

I would also change "Submit" to "Create client" text as its more clear in my opinion what the action is.

Reply:
@supalarry actually the form title and description should let a user know the purpose of the form and that is why imho its better to keep it as submit no?